dashboard/test/helpers/workshop_survey_results_helper_test.rb

```diff
@@ -107,6 +107,23 @@ class Pd::WorkshopSurveyResultsHelperTest < ActionView::TestCase
     assert_equal 3.67, result_hash[:how_clearly_presented]
   end
 
+  test 'averaging across multiple surveys' do
+    workshop_1 = create(:pd_workshop, :local_summer_workshop, num_sessions: 1, num_facilitators: 2, num_completed_surveys: 5)
+    workshop_2 = create(:pd_workshop, :local_summer_workshop, num_sessions: 1, num_facilitators: 3, num_completed_surveys: 10)
+
+    workshop_2.survey_responses.each do |response|
+      response.update_form_data_hash({how_clearly_presented: {workshop_2.facilitators.first.name => 'Not at all clearly', workshop_2.facilitators.second.name => 'Not at all clearly', workshop_2.facilitators.third.name => 'Not at all clearly'}})
```

Comment:
This is a really long line. Can we break it up to try to honor our 80 (120 max) [column width style](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#all-languages)?